src/angularjs/src/app/places/map/places-map.directive.js

```diff
@@ -0,0 +1,121 @@
+(function() {
+
+    /* @ngInject */
+    function PlacesMapController($filter, $http, $sanitize, $scope) {
+        var ctl = this;
+        ctl.map = null;
+        ctl.layerControl = null;
+
+        ctl.$onInit = function () {
+            ctl.mapOptions = {
+                scrollWheelZoom: true
+            };
+
+            // TODO: set center and zoom level by zooming to fit geojson polygon bounds
+            ctl.mapCenter = [39.963277, -75.142971];
+            ctl.baselayer = L.tileLayer(
+                'https://stamen-tiles.a.ssl.fastly.net/toner-lite/{z}/{x}/{y}.png', {
+                    attribution: 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="http://www.openstreetmap.org/copyright">ODbL</a>.',
+                    maxZoom: 18
+                });
+
+            $scope.$watch(function(){return ctl.pfbPlacesMapLayers;}, setLayers);
```

Comment:
Rather than setup a `$watch` here you should just be able to implement `ctl.$onChanges(changes)` if you switch pfbPlacesMapLayers to a one-way binding.

https://docs.angularjs.org/api/ng/service/$compile#life-cycle-hooks